# Assignment -1

In [27]:
# Create district 
dist = "Gwalior"

In [28]:
dist

'Gwalior'

#Variables- Containers for Storage

In [1]:
district = "Gwalior"
a=infected_till_date = 1234
b=infected_on_10thJune2020 = 12
c=infected_on_11thJune2020= 2
d=infected_on_12thJune2020 = 13

In [2]:
percentage_change_3days = ((b + c + d)/a)*100

In [3]:
print(percentage_change_3days)

2.188006482982172


# Report the % change in infected in three days is = 2.19%
